### PR TITLE
Fixed several avatar-related bugs

### PR DESCRIPTION
**Pull request by barankyle**
## Summary

Spawning of avatar now uses avatar object on user object. Client had previously been using fetchAvatarList and pulling the matching avatar off that, which was prone to failure if it was not on the first page. Instanceserver now gets user via feathers instead of sequelize model so that avatar is attached to user, and does not have to do a separate fetch of user's avatar.

Made project.update not ru onInstall hook if the project already exists in the database. This was causing avatars to be duplicated.

Updated fetchAvatarList to paginate. Updated avatar selection menus to paginate themselves by their page length and to make calls to fetchAvatarList when they need to get more from the API.

_A summary of changes being made in this PR_

## References

closes #_insert number here_

## Checklist
- [x] If this PR is still a WIP, convert to a draft
- [x] [ensure all checks pass](https://github.com/XRFoundation/XREngine/wiki/Testing-&-Contributing)
- [x] When this PR is ready, mark it as "Ready for review"
- [x] Changes have been manually QA'd
- [ ] Changes reviewed by at least 2 approved reviewer

## QA Steps

_List any additional steps required to QA the changes of this PR, as well as any supplemental images or videos._

### CODE REVIEW

File: packages/client-core/src/user/components/UserMenu/menus/EmoteMenu.tsx [hunk 1]
```diff
@@ -39,7 +39,7 @@ export const useEmoteMenuHooks = ({ changeActiveMenu }: EmoteMenuHooksProps) =>
     {
       body: <img src="/static/sad.svg" alt="sad" />,
       containerProps: {
-        onClick: () => runAnimation(AvatarStates.CLAP)
+        onClick: () => runAnimation(AvatarStates.DANCE2)
```
Comment (HexaField) [inline packages/client-core/src/user/components/UserMenu/menus/EmoteMenu.tsx:42]: why this change?

**Review by HexaField — CHANGES_REQUESTED** [review]

File: packages/client-core/src/user/components/UserMenu/menus/EmoteMenu.tsx (hunk 1 above)
Comment (barankyle) [reply packages/client-core/src/user/components/UserMenu/menus/EmoteMenu.tsx:42]: There'd been a comment about how it was confusing that the animation with the hand clapping icon wasn't the animation that actually clapped, so I swapped them. This admittedly isn't related to the other avatar fixes, I just threw it in. If there's a reason to the icon/animation pairing I overlooked, it can be reverted.

Comment (HexaField) [reply packages/client-core/src/user/components/UserMenu/menus/EmoteMenu.tsx:42]: no prob, i'll look into it
